CMakeLists.txt

```diff
@@ -366,6 +366,7 @@ option(WITH_RADOSGW_BEAST_FRONTEND "Rados Gateway's Beast frontend is enabled" O
 option(WITH_RADOSGW_BEAST_OPENSSL "Rados Gateway's Beast frontend uses OpenSSL" ON)
 option(WITH_RADOSGW_AMQP_ENDPOINT "Rados Gateway's pubsub support for AMQP push endpoint" ON)
 option(WITH_RADOSGW_KAFKA_ENDPOINT "Rados Gateway's pubsub support for Kafka push endpoint" ON)
+option(WITH_RADOSGW_LUA_SCRIPTING "Rados Gateway's support for Lua sripting" ON)
```

Comment:
can you think of a reason we'd need to disable this? it's not pulling in any new dependencies. i'd rather avoid the need for #ifdefs in rgw

Reply:
ok. will remove

Reply:
it looks like you removed the #ifdefs but not the option

Reply:
fixed now